src/ofdm.c

```diff
@@ -1099,13 +1099,14 @@ static int ofdm_sync_search_core(struct OFDM *ofdm) {
 
     PROFILE_SAMPLE_AND_LOG2(freq_start, "  freq");
 
-    if (ofdm->verbose != 0) {
+    if (ofdm->verbose > 1) {
         fprintf(stderr, "    ct_est: %4d foff_est: %4.1f timing_valid: %d timing_mx: %5.4f\n",
                 ct_est, (double) ofdm->coarse_foff_est_hz, timing_valid,
                 (double)timing_mx);
     }
 
-    if (timing_valid != 0) {
+    ofdm->timing_valid = timing_valid;
+    if (ofdm->timing_valid != 0) {
         /* potential candidate found .... */
```

Comment:
@srsampson :point_up: I think this is a bug in the OFDM modem -  `ofdm->timing_valid` wasn't being cleared when we transitioned from `synced` to `search`, resulting in an immediate transition back to `trial`, even if the acquisition conditions weren't valid.  Not too important at the end of a voice over I guess.